components/director/internal/model/document.go

```diff
@@ -56,3 +57,21 @@ func (d *DocumentInput) ToDocument(id, tenant, applicationID string) *Document {
 		Data:          d.Data,
 	}
 }
+
+func (d *DocumentInput) ToDocumentWithPackage(id, tenant string, packageID *string) *Document {
```

Comment:
What do you think about a name `ToDocumentWithinPackage`? And also please add TODO comment that eventually we will replace current `ToDocument` method with this logic.